scripts/meta/writeComComScripts.py

```diff
@@ -54,10 +54,10 @@
 
 location = 'slac' if len(sys.argv) < 2 else sys.argv[1]
 locationConfig = LocationConfig(location)
-butler = dafButler.Butler(locationConfig.ts8ButlerPath, collections=['LSST-TS8/raw/all', 'LSST-TS8/calib'])
+butler = dafButler.Butler(locationConfig.comCamButlerPath, collections=['LSSTComCam/raw/all', 'LSSTComCam/calib'])
```

Comment:
This commit https://github.com/lsst-sitcom/rubintv_production/pull/48/commits/2be66c1854878ea87eb96d15d9e96728ab757be2 should squash with https://github.com/lsst-sitcom/rubintv_production/pull/48/commits/7c5ee98ee4f48fd3eaa2555fe34857b9aaa3aa22